aiperf/common/config/input_config.py

```diff
@@ -218,6 +218,24 @@ def validate_dataset_type(self) -> Self:
         ),
     ] = InputDefaults.RANDOM_SEED
 
+    goodput: Annotated[
```

Comment:
@ajcasagrande @lkomali Is this the right config for goodput? This seems quite different to me than the other Input Config fields. It seems like something we'd want to be with other metrics or have its own config file.

Reply:
We want to make sure we're following the best path going forward. I don't want to carry over the same design decisions to AIPerf unless they make sense to do so.

I'm open to the argument that this is the best place for goodput to live in AIPerf, but given we're being more organized with the configs in AIPerf, this could be one of the args we could better organize.